sonar-html-plugin/src/main/java/org/sonar/plugins/html/node/Node.java

```diff
@@ -51,7 +49,7 @@ public int getEndLinePosition() {
   }
 
   public int getLinesOfCode() {
-    return StringUtils.countMatches(code, "\n");
+    return (int) getCode().chars().filter(charCode -> charCode == '\n').count();
```

Comment:
_minor_
```suggestion
    return (int) getCode().lines().count();
```

Reply:
Much better. Thanks !

Reply:
As discussed offline, this change would actually introduce a change in the underlying logic. While this would fix a wrong behavior, it's out of the scope of the current ticket.
I've created https://sonarsource.atlassian.net/browse/SONARHTML-165 to tackle this separately in our current sprint.